### DISCUSSION TITLE
Question: Consider open Github Discussion

### DISCUSSION THREAD

**Discussion by GeorgeS2019** [elsa-workflows/elsa-studio#19] [Ideas]
Being the first "professional" Blazor Workflow Designer, this is a 

# Game Changer

**Comment by GeorgeS2019** [comment]
Previous Blazor UI attempt for Elsa v2

# [Part 1 - UI driven by Elsa Workflows](https://medium.com/@fanekkie/part-1-ui-driven-by-elsa-workflows-6d0e868ea6c)
https://github.com/FransVanEk/ElsaDrivenUI

**Comment by sfmskywalker** [comment]
Good idea, Discussions is now enabled: https://github.com/elsa-workflows/elsa-studio/discussions

**Comment by sfmskywalker** [comment]
@GeorgeS2019 i think we can close this issue now that Discussions is enabled?
